internal/core/api.rs

```diff
@@ -324,6 +324,31 @@ pub enum SetRenderingNotifierError {
     AlreadySet,
 }
 
+/// This struct represents a persistent handle to a window and implements the traits
+/// for exposing raw window handles.
+#[cfg(feature = "raw-window-handle-06")]
+pub struct WindowHandle {
```

Comment:
I wonder if this should also derive Clone?

Reply:
yes it could.

Reply:
Done.